### PR TITLE
mount.ddi: add page

### PR DESCRIPTION
**Pull request by quantumflo**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by acuteenvy — CHANGES_REQUESTED** [review]
This is not an alias.

> **INVOCATION AS /SBIN/MOUNT.DDI**
> The systemd-dissect executable may be symlinked to /sbin/mount.ddi. If invoked through that it implements
> mount(8)'s "external helper" interface for the (pseudo) file system type "ddi". This means
> conformant disk images may be mounted directly via
>```
> # mount -t ddi myimage.raw targetdir/
>```
> in a fashion mostly equivalent to:
>```
> # systemd-dissect --mount myimage.raw targetdir/
>```

`mount.ddi --mount myimage.raw targetdir/` doesn't work. The equivalent is `mount.ddi myimage.raw targetdir/`.

File: pages/linux/mount.ddi.md
Comment (acuteenvy) [inline pages/linux/mount.ddi.md:8]:
```suggestion
# mount.ddi

> Mount Discoverable Disk Images.
> See `tldr systemd-dissect` for other commands relevant to DDIs.
> More information: <https://www.freedesktop.org/software/systemd/man/latest/systemd-dissect.html>.

- Mount an OS image:

`mount.ddi {{path/to/image.raw}} {{/mnt/image}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, after @acuteenvy's suggestion.

**Review by acuteenvy — APPROVED** [review]

**Comment by quantumflo** [comment]
@kbdharun Please re add me as maintainer. I was removed due to 2 factor authentication

**Comment by kbdharun** [comment]
> @kbdharun Please re add me as maintainer. I was removed due to 2 factor authentication not being enabled

Done, I have sent an invitation.

**Pull request merged**
